### PR TITLE
Upgraded static files to Tinymce v5

### PR DESCRIPTION
**Pull request by urtzai**
This pull request upgrades the Tiny editor to the last version (v5)

### CODE REVIEW

File: tinymce/static/tiny_mce/langs/ar.js [hunk 1]
```diff
@@ -1 +0,0 @@
...u0631","339966":"\u0627\u062e\u0636\u0631 \u0628\u062d\u0631\u064a","33CCCC":"\u0641\u064a\u0631\u0648\u0632\u064a","3366FF":"\u0627\u0644\u0623\u0632\u0631\u0642 \u0627\u0644\u0645\u0644\u0643\u064a","800080":"\u0623\u0631\u062c\u0648\u0627\u0646\u064a","999999":"\u0631\u0645\u0627\u062f\u064a \u0645\u062a\u0648\u0633\u0637",FF00FF:"\u0641\u0648\u0634\u064a",FFCC00:"\u0630\u0647\u0628\u064a",FFFF00:"\u0623\u0635\u0641\u0631","00FF00":"\u0644\u064a\u0645\u0648\u0646\u064a","00FFFF":"\u0645\u0627\u0626\u064a","00CCFF":"\u0633\u0645\u0627\u0648\u064a","993366":"\u0628\u0646\u064a",C0C0C0:"\u0641\u0636\u064a",FF99CC:"\u0632\u0647\u0631\u064a",FFCC99:"\u062e\u0648\u062e\u064a",FFFF99:"\u0623\u0635\u0641\u0631 \u0645\u0636\u064a\u0621",CCFFCC:"\u0623\u062e\u0636\u0631 \u0634\u0627\u062d\u0628",CCFFFF:"\u0633\u0645\u0627\u0648\u064a \u0634\u0627\u062d\u0628","99CCFF":"\u0633\u0645\u0627\u0648\u064a \u0645\u0636\u064a\u0621",CC99FF:"\u0628\u0646\u0641\u0633\u062c\u064a",FFFFFF:"\u0623\u0628\u064a\u0636"},aria:{"rich_text_area":"\u0645\u0633\u0627\u062d\u0629 \u0627\u0644\u062a\u0639\u062f\u064a\u0644"},wordcount:{words:"\u0639\u062f\u062f \u0627\u0644\u0643\u0644\u0645\u0627\u062a :"}}});
```
Comment (Natim) [inline tinymce/static/tiny_mce/langs/ar.js]: Why do you keep those files if they now are empty?

Comment (urtzai) [reply tinymce/static/tiny_mce/langs/ar.js]: Sorry it's a mistake. Need to delete them, as well as we need to correct the tests.

Comment (urtzai) [reply tinymce/static/tiny_mce/langs/ar.js]: Hi Natim, I've doubted they weren't deleted but they already are...

Comment (Natim) [reply tinymce/static/tiny_mce/langs/ar.js]: Ok I guess that's a bug in @Github then.

**Comment by urtzai** [comment]
Not sure if "tinymce_preview" template tag is needed anymore...

**Pull request merged**